src/Apps/Artist/Components/__tests__/ArtistConsignButton.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { Breakpoint } from "@artsy/palette"
+import { ArtistConsignButtonQueryRawResponse } from "__generated__/ArtistConsignButtonQuery.graphql"
+import { MockBoot, renderRelayTree } from "DevTools"
+import React from "react"
+import { graphql } from "react-relay"
+import { ArtistConsignButtonFragmentContainer } from "../ArtistConsignButton"
+
+jest.unmock("react-relay")
+
+describe("ArtistConsignButton", () => {
+  const getWrapper = async ({
+    breakpoint = "xs",
+    response,
+  }: {
+    breakpoint: Breakpoint
+    response: ArtistConsignButtonQueryRawResponse
+  }) => {
+    return await renderRelayTree({
+      Component: ({ artist }) => {
+        return (
+          <MockBoot breakpoint={breakpoint}>
+            <ArtistConsignButtonFragmentContainer artist={artist} />
+          </MockBoot>
+        )
+      },
+      query: graphql`
+        query ArtistConsignButton_Test_Query($artistID: String!)
+          @raw_response_type {
+          artist(id: $artistID) {
+            ...ArtistConsignButton_artist
+          }
+        }
+      `,
+      variables: {
+        artistID: "alex-katz",
+      },
+      mockData: response,
+    })
+  }
+
+  describe("Top 20 Button", () => {
+    const response = {
+      artist: {
+        name: "Alex Katz",
+        href: "/artist/alex-katz",
+        image: {
+          cropped: {
+            url:
+              "https://d7hftxdivxxvm.cloudfront.net?resize_to=fill&width=75&height=66&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FbrHdWfNxoereaVk2VOneuw%2Flarge.jpg",
+          },
+        },
+        id: "QXJ0aXN0OjRkOGQxMjBjODc2YzY5N2FlMTAwMDA0Ng==",
+      },
+    }
+
+    it("renders desktop version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "md", response })
+      expect(wrapper.find("Image").length).toEqual(1)
+      expect(wrapper.text()).toContain("Sell your Alex Katz")
+      expect(wrapper.find("RouterLink").html()).toContain(
+        `href="/artist/alex-katz/consign"`
+      )
+    })
+
+    it("renders mobile version", async () => {
+      const wrapper = await getWrapper({ breakpoint: "xs", response })
+      expect(wrapper.find("Image").length).toEqual(1)
+      expect(wrapper.text()).toContain("Sell your Alex Katz")
+      expect(wrapper.find("RouterLink").html()).toContain(
+        `href="/artist/alex-katz/consign"`
+      )
```

Comment:
These tests are rendering with different breakpoints, but because the assertions are identical, there's not much safety that we're rendering `xs` differently from `>xs`. I think this is because there are *aren't* any significant differences between the breakpoints, other than layout implementation details. Am I correct in my understanding?

Reply:
There's two different components triggered by breakpoints, so we want to ensure that both work as expected.